region/lowernorfair/east.json

```diff
@@ -2205,14 +2217,31 @@
           "to": [
             {
               "id": 2,
-              "requires": null,
+              "requires": [
+                {"or": [
+                  "canWalljump",
+                  "SpaceJump"
```

Comment:
There are so many alternatives to walljump besides space jump (Successively climbing on two frozen firefleas, midair springball jump, ibj). I don't mind walljump being there only sporadically as a req for now, but when it's put in it should be fully put in.

Reply:
Ok I'm stupid. The Firefleas won't work for the second portion :P

Reply:
Actually hjb + Speed Booster also works

Reply:
Adding canUseFrozenEnemies between nodes 4 & 5. Adding HiJ+SpeedBooster for 5 to 2.

Reply:
You should also add hjb + springBallJumpMidAir, and replace Space Jump with canFly.

Reply:
Done!